package.json

```diff
@@ -91,6 +91,7 @@
     "postcss-dir-pseudo-class": "^6.0.0",
     "postcss-logical": "^5.0.0",
     "pre-commit": "^1.2.2",
+    "process": "^0.11.10",
```

Comment:
```suggestion
```

Reply:
This is automatically added by parcel whenever it starts up. We have some weird beta versions running. It think this can be solved by upgrading them.